### DISCUSSION TITLE
Wayback machine

### DISCUSSION THREAD

**Discussion by jcubic** [facebook/docusaurus#10293] [General] [answered]
Do you have any experience with Wayback Machine? It would be nice to have a working website when it's cloned by Archive.org. It looks like the Docusaurus website is totally broken (because of baseURL, I hope this is the only issue):

https://web.archive.org/web/20240423084519/https://lips.js.org/

**Comment by slorber** [comment, marked as answer]
Looks like it's working as intended?

https://web.archive.org/web/20240101151623/https://docusaurus.io/

https://web.archive.org/web/20240607172942/https://lips.js.org/

Maybe the site was actually broken at the given snapshot time you selected? 🤷‍♂️
